exasol/nb_connector/cli/param_wrappers.py

```diff
@@ -0,0 +1,113 @@
+"""
+Wrappers for adding custom properties to click parameters, e.g. SCS key.
+"""
+
+import click
+
+from exasol.nb_connector.ai_lab_config import AILabConfig as CKey
+
+
+class ScsArgument:
+    """
+    Represents a CLI argument for the SCS command.
+    """
+
+    def __init__(self, *args, scs_key: CKey | None = None, **kwargs):
+        self._args = args
+        self.scs_key = scs_key
```

Comment:
how do you plan to use this? inside the function call, you only get the value of argument or option, but not this scs_key